paket.dependencies

```diff
@@ -31,4 +31,5 @@ nuget FsCheck.Xunit
 nuget Machine.Specifications.Should
 nuget Machine.Specifications.Runner.Console
 nuget Nancy.Testing
-nuget xunit.extensions
\ No newline at end of file
+nuget xunit.extensions
+nuget Newtonsoft.Json
```

Comment:
I was so happy that I got rid of NewtonSoft.Json ;-)

Seems it coming back over and over again.